docs/hello-signing.md

```diff
@@ -72,48 +72,54 @@ ACME Rockets will only deploy software that's been scanned and approved by the A
 - List the image, and any associated signatures
 
   ```bash
-  notation list --plain-http $IMAGE
+  notation list $IMAGE
   ```
 
   At this point, the results are empty, as there are no existing signatures
 
 ## Signing a Container Image
 
-To get things started quickly, the Notation cli supports generating self signed certificates. As you automate the signing of content, you will most likely want to create and store the private keys in a key vault. (Detailed production steps will be covered later)
+To get things started quickly, the Notation cli supports self-generated certificate chain. As you automate the signing of content, you will most likely want to create and store the private keys in a key vault. (Detailed production steps will be covered later)
 
-- Generate a self-signed test certificate for signing artifacts
-  The following will generate a self-signed X.509 certificate under the `~/config/notation/` directory
+- Create a self-generated test certificate chain for signing artifacts.
+  The following will create a self-generated X.509 certificate chain under the `~/config/notation/localkeys` directory along with a private key.
 
   ```bash
   notation cert generate-test --default "wabbit-networks.io"
   ```
 
-- Sign the container image
+- Sign the container image. By default, JWS envelope is used.
+  On success, the signature envelope will be cached under the `~/cache/notation/signatures` directory.
 
   ```bash
-  notation sign --plain-http $IMAGE
+  notation sign $IMAGE
+  ```
+  To sign with COSE envelope
+  ```bash
+  notation sign --envelope-type cose $IMAGE
```

Comment:
@yizha1 @patrickzheng200 @shizhMSFT  Can we provide a system configuration option for signature envelop? In this way, users don't need to manually add this tag each time.

Reply:
@chloeyin adding Zaihao into this discussion.